gatsby-node.js

```diff
@@ -305,8 +306,11 @@ const localFileCheck = async (
       // the cleanKey here
       data[key].forEach((attachment) => {
         const ext = mime.getExtension(attachment.type); // unknown type returns null
+        // get the filename from the airtable metadata instead of the remote file
+        const airtableFileName = path.parse(attachment.filename).name;
         let attachmentNode = createRemoteFileNode({
           url: attachment.url,
+          name: airtableFileName,
```

Comment:
```suggestion
          name: airtableFile.name.replace(/[/\\?%*:|"<>]/g, ""),
```

Since we are using the filename from Airtable directly, that may contain illegal filename characters. This is primarily a Windows concern (hence why that failed). If we aren't relying on `gatsby-source-filesystem` to handle the filename based on the url, we now need to handle this case. This seems to be why it was failing in CI only on windows.